nuxt.config.ts

```diff
@@ -44,4 +39,10 @@ export default defineNuxtConfig({
   },
   plugins: [{ src: "~/plugins/vercel.ts", mode: "client" }],
   css: ["vue-final-modal/style.css"],
+  routeRules: {
+    "/": { isr: true },
+    "/api/**": { cors: true, isr: 60 },
+    "/contributing": { isr: true },
+    "/list": { isr: true },
+  },
```

Comment:
From https://nuxt.com/docs/guide/concepts/rendering#route-rules:
* cors: boolean - Automatically adds cors headers with cors: true - you can customize the output by overriding with headers
* isr: number | boolean - The behavior is the same as swr except that we are able to add the response to the CDN cache on platforms that support this (currently Netlify or Vercel). If true is used, the content persists until the next deploy inside the CDN.

This configuration caches pages until the next deployment and caches API routes for one minute. Also adds CORS headers.